addons/metadata.demo.movies/demo.py

```diff
@@ -0,0 +1,116 @@
+    liz=xbmcgui.ListItem('Demo movie 2', thumbnailImage='DefaultVideo.png', offscreen=True)
+    liz.setProperty('relevance', '0.3')
+    xbmcplugin.addDirectoryItem(handle=int(sys.argv[1]), url="/path/to/movie2", listitem=liz, isFolder=True)
+elif action == 'getdetails':
+    url=urllib.unquote_plus(params["url"])
+    if url == '/path/to/movie':
+        liz=xbmcgui.ListItem('Demo movie 1', offscreen=True)
+        liz.setProperty('video.original_title', 'Demo måvie 1')
+        liz.setProperty('video.sort_title', '2')
+        liz.setProperty('video.ratings', '1')
+        liz.setProperty('video.rating1.value', '5')
+        liz.setProperty('video.rating1.votes', '100')
+        liz.setProperty('video.user_rating', '5')
+        liz.setProperty('video.top250', '3')
+        liz.setProperty('video.unique_id', '123')
+        liz.setProperty('video.imdb_id', '456')
+        liz.setProperty('video.plot_outline', 'Outline yo')
+        liz.setProperty('video.plot', 'Plot yo')
+        liz.setProperty('video.tag_line', 'Tag yo')
+        liz.setProperty('video.duration_minutes', '110')
+        liz.setProperty('video.mpaa', 'T')
+        liz.setProperty('video.trailer', '/home/akva/Videos/porn/bukkake.mkv')
+        liz.setProperty('video.thumbs', '2')
+        liz.setProperty('video.thumb1.url', 'DefaultBackFanart.png')
+        liz.setProperty('video.thumb1.aspect', 'poster')
+        liz.setProperty('video.thumb2.url', '/home/akva/Pictures/hawaii-shirt.png')
+        liz.setProperty('video.thumb2.aspect', 'banner')
+        liz.setProperty('video.genre','Action / Comedy')
+        liz.setProperty('video.country', 'Norway / Sweden / China')
+        liz.setProperty('video.writing_credits', 'None / Want / To Admit It')
+        liz.setProperty('video.director', 'spiff / spiff2')
+        liz.setProperty('video.tvshow_links' ,'Demo show 1')
+        liz.setProperty('video.actors', '2')
+        liz.setProperty('video.actor1.name', 'spiff')
+        liz.setProperty('video.actor1.role', 'himself')
+        liz.setProperty('video.actor1.sort_order', '2')
+        liz.setProperty('video.actor1.thumb', '/home/akva/Pictures/fish.jpg')
+        liz.setProperty('video.actor1.thumb_aspect', 'banner')
+        liz.setProperty('video.actor2.name', 'monkey')
+        liz.setProperty('video.actor2.role', 'orange')
+        liz.setProperty('video.actor2.sort_order', '1')
+        liz.setProperty('video.actor1.thumb_aspect', 'poster')
+        liz.setProperty('video.actor2.thumb', '/home/akva/Pictures/coffee.jpg')
+        liz.setProperty('video.set_name', 'Spiffy creations')
+        liz.setProperty('video.set_overview', 'Horrors created by spiff')
+        liz.setProperty('video.tags', 'Very / Bad')
+        liz.setProperty('video.studio', 'Studio1 / Studio2')
+        liz.setProperty('video.fanarts', '2')
+        liz.setProperty('video.fanart1.url', 'DefaultBackFanart.png')
```

Comment:
Dont follow.  All fanarts are stored based on the cfanart member afaik.
16. jun. 2016 09:37 skrev "Philipp Temminghoff" <notifications@github.com>:

> In addons/metadata.demo.movies/demo.py
> <https://github.com/xbmc/xbmc/pull/9984#discussion_r67299282>:
>
> > +        liz.setProperty('video.actor1.name', 'spiff')
> > +        liz.setProperty('video.actor1.role', 'himself')
> > +        liz.setProperty('video.actor1.sort_order', '2')
> > +        liz.setProperty('video.actor1.thumb', '/home/akva/Pictures/fish.jpg')
> > +        liz.setProperty('video.actor1.thumb_aspect', 'banner')
> > +        liz.setProperty('video.actor2.name', 'monkey')
> > +        liz.setProperty('video.actor2.role', 'orange')
> > +        liz.setProperty('video.actor2.sort_order', '1')
> > +        liz.setProperty('video.actor1.thumb_aspect', 'poster')
> > +        liz.setProperty('video.actor2.thumb', '/home/akva/Pictures/coffee.jpg')
> > +        liz.setProperty('video.set_name', 'Spiffy creations')
> > +        liz.setProperty('video.set_overview', 'Horrors created by spiff')
> > +        liz.setProperty('video.tags', 'Very / Bad')
> > +        liz.setProperty('video.studio', 'Studio1 / Studio2')
> > +        liz.setProperty('video.fanarts', '2')
> > +        liz.setProperty('video.fanart1.url', 'DefaultBackFanart.png')
>
> Wouldnt we need a 2nd prefix in order to indicate that something should
> get written into the art table? (so sth like "video.art.fanart" in this
> case)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/xbmc/pull/9984/files/94362ac88617f4c7cd9157aff6bbbd22cd3b30ea#r67299282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFktYDUUQu_PIuw5QSzAEspjhAZUXYDfks5qMP1GgaJpZM4I2zXQ>
> .
>